bibliopixel/project/project.py

```diff
@@ -76,23 +82,42 @@ def resolve_aliases(name):
 
     if not (driver or drivers):
         raise ValueError(
-            'The project has neither a "driver" nor a "drivers" sections')
+            'The project has neither a "driver" nor a "drivers" section')
 
     extend_path(path)
     maker = data_maker.Maker(**(maker or {}))
     make_object = functools.partial(importer.make_object, maker=maker)
 
-    builder = MultiMapBuilder(make_object)
-    driver_objects = builder.make_drivers(driver, drivers)
+    driver_objects = make_drivers(driver, drivers, make_object)
+
+    gen_coord_map = layout.pop('gen_coord_map', None)
```

Comment:
IF we're going to expose this name, it has to be a lot better than `gen_coord_map`!  This name raises almost as many questions as it answers....

What in English does it do, exactly?

What is a `coord_map`, exactly?